### ISSUE TITLE
Intergrate with SIGMA

### ISSUE THREAD

**Issue by rhaist** [0xrawsec/whids#8]
As SIGMA is already an open source log signature standard adopted by many tools it might be beneficial to integrate it with whids. As an instant benefit the already tested SIGMA detection rules can be used.

REF: https://github.com/Neo23x0/sigma

**Comment by qjerome** [comment]
Hi @rhaist,

Translating 1 to 1 Sigma to Gene is not very straightforward. This can be explained by the fact that Gene format is regex based and databases like Splunk, Arcsight Loggers and Elastic Search are text based. By extensions most of the Sigma rules are also written with text based search syntax in mind. I did write an [export tool](https://github.com/0xrawsec/gene/blob/master/scripts/sigma2gene.py) to port Sigma rules to Gene rules a while ago. Even though it is not impossible to design a flawless transformation between the two formats, I don't claim I was able to do it. That is why I would not recommend to use Sigma rules to feed WHIDS without further manual verification or testing. The purpose of Sigma is to provide a translation from Sigma format to other formats so I think that a reliable translation from Sigma to Gene should be addressed in the Sigma project rather than in this one.

Cheers,
